{{ cookiecutter.repo_name }}/.envrc

```diff
@@ -1,52 +1,23 @@
-# Environment variables go here, and can be read in by Python using `os.getenv`:
+# Orchestration file to load environment variables from the `.env` and `.secrets` files.
 #
-#   --------------------------------------------------------
+# Only used by systems with `direnv` (https://direnv.net/) installed. Environment
+# variables can be read in by Python using `os.getenv` _without_ using `python-dotenv`:
+#
+#   ------------------------------------------------------------------------------------
 #   import os
 #
 #   # Example variable
 #   EXAMPLE_VARIABLE = os.getenv("EXAMPLE_VARIABLE")
-#   --------------------------------------------------------
-#
-# To ensure the `sed` command below works correctly, make sure all file paths in environment variables are absolute
-# (recommended), or are relative paths using other environment variables (works for Python users only). Environment
-# variable names are expected to contain letters, numbers or underscores only.
+#   ------------------------------------------------------------------------------------
 #
-# DO NOT STORE SECRETS HERE - this file is version-controlled! You should store secrets in a `.secrets` file, which is
-# not version-controlled - this can then be sourced here, using `source_env ".secrets"`.
-
-# Extract the variables to `.env` if required. Note `.env` is NOT version-controlled, so `.secrets` will not be
-# committed
-sed -n 's/^export \(.*\)$/\1/p' .envrc .secrets | sed -e 's?$(pwd)?'"$(pwd)"'?g' | sed -e 's?$\([a-zA-Z0-9_]\{1,\}\)?${\1}?g' > .env
+# DO NOT STORE SECRETS HERE - this file is version-controlled! You should store secrets
+# in the untracked `.secrets` file. This is loaded here using the `dotenv_if_exists`
+# command.
 
-# Add the working directory to `PYTHONPATH`; allows Jupyter notebooks in the `notebooks` folder to import `src`
+# Add the working directory to `PYTHONPATH`; allows Jupyter notebooks in the
```

Comment:
Why is this done here whilst everything else is now done in .env? Does this work for Jupyter notebooks users on windows?

Reply:
So this sets an environment variable for linux but to do this on windows you have to go into your environment variables. To install src on a notebook though you can just change the working directory so this functionality is still there you just don't set an environment variable for it.